business_requirement_etherpad/README.rst

```diff
@@ -0,0 +1,166 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+
+=============================
+Business Requirement Etherpad
+=============================
+
+
+Configuration
```

Comment:
"This module extends the functionality of the business requirements and introduces a field link to Etherpad in order for the team to keep internal notes about the BR"

Remove configuration and Usage content.